service/mantle/ups/UpsServices.xml

```diff
@@ -47,4 +47,242 @@
         </actions>
     </service>
 
+    <!-- ========== Preparing Shipment Map for Label Service ========== -->
+    <service verb="get" noun="ShipmentRequestMap">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
```

Comment:
Please recheck this as per the comment on https://github.com/hotwax/mantle-ups/pull/8/files#r786222889. 
The input parameters will differ, you can also refer mantle shippo component.